roles/ckan/templates/kubernetes/ckandb_job.yaml

```diff
@@ -24,7 +24,9 @@ spec:
               value: "{{ ckan_db_hostname }}"
             - name: POSTGRES_USER
               value: "{{ rds_admin_username }}"
-            
+            - name: POSTGRES_PASSWORD
+              value: "{{ ckan_postgres_password }}"
+ 
```

Comment:
Don't understand why this is needed?

Reply:
postgresdb pod fails with authentication error as it requires password on local deployment, which got accidentally removed 
[d7a722bb3e5cfde6179d410f9d287152fc8c09d4290b675a7aac57369fb55a2dR28](https://github.com/fjelltopp/fjelltopp-ansible/pull/1/files#diff-d7a722bb3e5cfde6179d410f9d287152fc8c09d4290b675a7aac57369fb55a2d)